lnme.go

```diff
@@ -188,7 +188,15 @@ func main() {
 	if os.Getenv("PORT") != "" {
 		port = os.Getenv("PORT")
 	}
-	e.Logger.Fatal(e.Start(":" + port))
+	listen := cfg.String("listen")
+	if os.Getenv("LISTEN") != "" {
```

Comment:
isn't this done by the https://github.com/knadh/koan loadConfig already?
if you do `LNME_LISTEN` ? (https://github.com/bumi/lnme/blob/master/lnme.go#L217) is that an option for you? 

I think I had the `PORT` check there because some cloud hoster set the `PORT` environment variable. (should add a comment there)

Reply:
`LNME_LISTEN` is no problem for me, wanted to make the least disruptive change (leaving everything similar to `PORT`)